test/basic.js

```diff
@@ -133,7 +133,7 @@ test('truncate event has truncated-length and fork', async function (t) {
   await core.truncate(2)
 })
 
-test('treeHash gets the tree hash at a given core length', async (t) => {
```

Comment:
This was me being inconsistent. There doesn't happen to be a Standard option for this?